pgoapi/pgoapi.py

```diff
@@ -396,12 +394,10 @@ def spin_nearest_fort(self):
             self.log.info('Nearest fort distance is %s', nearest_fort_dis)
 
             # Fort is close enough to change our route and walk to
-            if nearest_fort_dis > 40.00 and nearest_fort_dis <= 100:
-                lat = nearest_fort['latitude']
-                long = nearest_fort['longitude']
-                self.walk_to_fort(destinations[0], directly=True)
-                self.fort_search_pgoapi(nearest_fort, player_postion=self.get_position(),
-                                        fort_distance=nearest_fort_dis)
+            if self.wander_steps:
+                if nearest_fort_dis > 40.00 and nearest_fort_dis <= self.wander_steps:
+                    self.walk_to_fort(destinations[0], directly=True)
+
```

Comment:
since we walked to the nearest fort, shouldn't we spin it as well? maybe update the nearest fort distance and let te next if statement handle it.